src/apps/components/AppFrame/useAppActions.ts

```diff
@@ -3,11 +3,18 @@ import { usePostToExtension } from "@dashboard/apps/components/AppFrame/usePostT
 import { Actions, DispatchResponseEvent } from "@saleor/app-sdk/app-bridge";
 import React, { useState } from "react";
 
+/**
+ * TODO Refactor to named attributes
+ */
```

Comment:
suggestion: leaving todo in code means nothing, wouldn't it be better to create an issue for this and link it here?

Reply:
From my experience TODO comments in code are easier to track and fix, because IDEs track them and I see them every time near the code.

If I create an issue in the backlog for every small case like this, they will be forever gone in the backlog.

Here, I want to continue with refactoring src/apps so I left myself comments for future places I should refactor, so I don't mix refactors with features like here